### PR TITLE
Create i-marsh.md

### LINKED ISSUE #3: Need to add Ian as a participant
Add new participant to participant folder with _name, handle and favorite color_
cc @mfilosa

### PR DESCRIPTION
**Pull request by i-marsh**
adds Ian Marsh as a participant in the workshop
cc: @mfilosa @nathos ... please review and feedback... approve if poss
resolves #3

### CODE REVIEW

File: Participants/i-marsh.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Hi, I'm i-marsh
+
+| Property | Value |
+|:---------------|:-----------------|
+| Name | Ian Marsh |
+| Favorite color | Green |
```
Comment (nathos) [inline Participants/i-marsh.md]: I know for a fact that green _is not_ your favorite color. Could you update this please? 

elf-throneoflies

**Review by nathos — CHANGES_REQUESTED** [review]
Please take a look at this feedback and resolve!

File: Participants/i-marsh.md (hunk 1 above)
Comment (i-marsh) [reply Participants/i-marsh.md]: I have no idea what my favorite color is ... especially when it's **_colour_**

**Review by nathos — APPROVED** [review]

**Comment by i-marsh** [comment]
ok ✔️ done... closes #3

**Pull request merged**